operator/v1/types_etcd.go

```diff
@@ -28,12 +28,33 @@ type Etcd struct {
 
 type EtcdSpec struct {
 	StaticPodOperatorSpec `json:",inline"`
+	// profile allows user to change the etcd tuning profile which configures
+	// the latency parameters for heartbeat interval and leader election timeouts
+	// allowing the cluster to tolerate longer round-trip-times between etcd members.
+	// Valid values are "Default" and "Slower".
+	// The current default is Default.
+	// +kubebuilder:validation:Optional
+	// +openshift:enable:FeatureSets=CustomNoUpgrade;TechPreviewNoUpgrade
+	// +optional
+	Profile ProfileValue `json:"profile,omitempty"`
 }
 
 type EtcdStatus struct {
 	StaticPodOperatorStatus `json:",inline"`
 }
 
+const (
+	// DefaultProfile sets Heartbeat interval: 100ms, Leader election timeout: 1000ms
+	DefaultProfile ProfileValue = "Default"
+	// SlowerProfile sets Heartbeat interval: 500ms, Leader election timeout: 2500ms
+	SlowerProfile ProfileValue = "Slower"
+)
+
+// profileValue declares valid etcd tuning profiles
+// +enum
+// +kubebuilder:validation:Enum:="Default";"Slower"
+type ProfileValue string
```

Comment:
Would ProfileValue -> Profile work as well? A `Profile` itself can be used as a field name.

Reply:
I'm not sure I understand. Are you suggesting to rename the ProfileValue type to just Profile?

Reply:
yes